runtime/src/lib.rs

```diff
@@ -320,8 +317,6 @@ construct_runtime!(
 		Balances: pallet_balances,
 		TransactionPayment: pallet_transaction_payment,
 		Sudo: pallet_sudo,
-		// Include the custom logic from the pallet-template in the runtime.
-		TemplateModule: pallet_template,
```

Comment:
In the future when we create a new pallet, do you think having a template already would have been useful or not? I kept it because I thought it would save us time to create all the files manually.

Reply:
I mean in a code which may go through some audits, external reviews it is not good to keep template code. A substrate developer should be able to create all files required for any new pallets :smile: